fcrepo-stats-api/src/main/java/org/fcrepo/stats/api/BinaryStatsResult.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.stats.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ *  A data object representing a statistics result for binary objects
+ * @author dbernstein
+ */
+public class BinaryStatsResult extends StatsResult {
+
+    @JsonProperty("byte_count")
+    private Long byteCount = -1l;
```

Comment:
Any strong reason why this is initialized to -1 while StatsResult is initialized to 0?

Reply:
good catch.